docs/docs/mdx/index.md

```diff
@@ -2,7 +2,7 @@
 title: Adding Components to Content with MDX
 ---
 
-When writing long-form content in Markdown you might want to embed components.
+When writing long-form content in Markdown you might want to embed [components](/docs/glossary/#components).
```

Comment:
One correction: I think the glossary page was just edited today or yesterday to be singular.
 
```suggestion
When writing long-form content in Markdown you might want to embed [components](/docs/glossary/#component).
```

Reply:
Good catch! Looks like it changed today: https://github.githistory.xyz/gatsbyjs/gatsby/blob/master/docs/docs/glossary.md